Solutions/Corvus.Extensions.CosmosClient.Specs/Common/Person.cs

```diff
@@ -41,9 +41,9 @@ public bool Equals(Person other)
         public override int GetHashCode()
         {
             int hashCode = -605153749;
-            hashCode = (hashCode * -1521134295) + EqualityComparer<string>.Default.GetHashCode(this.Name);
+            hashCode = (hashCode * -1521134295) + (this.Name == null ? 0 : EqualityComparer<string>.Default.GetHashCode(this.Name));
```

Comment:
I'm using `0` as for the hashcode segment when property is null. Are we happy with this?